resources/js/Pages/ModPacks/Components/EditModPackForm.vue

```diff
@@ -0,0 +1,271 @@
+                        <multi-select
+                                v-model="form.servers"
+                                :options-list="serversOptions"
+                                :tags="true"
+                                placeholder="Chose a server"
+                        />
+                    </div>
+                </template>
+            </monolithic-form-input-card>
+        </template>
+
+        <template #actions>
+            <jet-action-message
+                    :on="form.recentlySuccessful"
+                    class="mr-3"
+            >
+                {{ formSuccessMsg }}
+            </jet-action-message>
+
+            <jet-secondary-button
+                    @click="goToUpdate"
+                    :class="{ 'opacity-25': form.processing }"
+                    :disabled="form.processing"
+                    class="mr-3"
+                    type="button"
+            >
+                Manage Updates
+            </jet-secondary-button>
+
+            <jet-button
+                    :class="{ 'opacity-25': form.processing }"
+                    :disabled="form.processing"
+                    class="mr-3"
+                    type="submit"
+            >
+                Save
+            </jet-button>
+        </template>
+    </monolithic-form-section>
+</template>
+
+<script lang="ts">
+import { Component, Mixins, Prop } from 'vue-property-decorator'
+import MonolithicFormSection from '@/Shared/Forms/MonolithicFormSection.vue'
+import MonolithicFormInputCard from '@/Shared/Forms/MonolithicFormInputCard.vue'
+import JetLabel from '@/Jetstream/Label.vue'
+import JetInput from '@/Jetstream/Input.vue'
+import JetInputError from '@/Jetstream/InputError.vue'
+import JetActionMessage from '@/Jetstream/ActionMessage.vue'
+import JetButton from '@/Jetstream/Button.vue'
+import JetSecondaryButton from '@/Jetstream/SecondaryButton.vue'
+import JetSectionBorder from '@/Jetstream/SectionBorder.vue'
+import JetSectionTitle from '@/Jetstream/SectionTitle.vue'
+import JetDropdown from '@/Jetstream/Dropdown.vue'
+import MultiSelect from '@/Shared/Forms/MultiSelect.vue'
+import { MultiSelectOptions } from '@/Shared/Forms/Types/MultiSelectOptions'
+
+import Route from '@/Mixins/Route'
+import Required from '@/Shared/Forms/Required.vue'
+
+import MultiSelectServerRow from '@/Shared/Forms/MultiSelectServerRow.vue'
+
+@Component({
+  components: {
+    MonolithicFormSection,
+    MonolithicFormInputCard,
+    Required,
+    MultiSelect,
+    JetLabel,
+    JetInput,
+    JetInputError,
+    JetActionMessage,
+    JetButton,
+    JetSecondaryButton,
+    JetSectionBorder,
+    JetSectionTitle,
+    JetDropdown,
+  },
+})
+export default class EditModPackForm extends Mixins(Route) {
+  @Prop({
+    type: Object,
+    required: true,
+  }) modpack!: any
+
+  @Prop() readonly servers !: Array<any>
+
+  serversOptions: MultiSelectOptions = this.initServerOptions()
+
+  form = this.$inertia.form({
+    _method: 'PUT',
```

Comment:
Retirer  ` _method: 'PUT',`
Changer `this.$inertia.form` par `this.$inertia.put`